orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/clone/CloneServerGroupTask.groovy

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.netflix.spinnaker.orca.clouddriver.tasks.servergroup
+package com.netflix.spinnaker.orca.clouddriver.tasks.servergroup.clone
```

Comment:
@jkschneider : any reason why this was changed to a different package? This breaks the upgrade scenario when you have a bunch of messages in the queue with `"taskType": "com.netflix.spinnaker.orca.clouddriver.tasks.servergroup.CloneServerGroupTask",`
and you upgrade to this orca (same for downgrade)

do we have a mechanism to coerce types that I can fix before moving this back to the old package?

(cc: @ajordens)

Reply:
Looking and I think we can use `SerializationMigrator` to fix this up

Reply:
@jkschneider FYI: i am going to move this back as the migrator is not a good solution (since we will not be able to roll back from it if need be). I will CC you on the PR, shortly